static/app/utils/profiling/renderers/flamegraphRenderer.tsx

```diff
@@ -83,12 +83,11 @@ class FlamegraphRenderer {
   }
 
   init(): void {
-    const VERTICES = 6;
+    const VERTICES_PER_TRIANGLE = 6;
```

Comment:
```suggestion
    const VERTICES_PER_FRAME = 6;
```

And this constant is declared multiple times throughout this file. Perhaps consider lifting it to the top?